### PR TITLE
MGDSTRM-8261-Updating Admin server to use Quarkus 2.7.5 and smallrye-jwt 3.5.3

### PR DESCRIPTION
**Pull request by biswassri**

### CODE REVIEW

File: kafka-admin/pom.xml [hunk 1]
```diff
@@ -238,6 +234,11 @@
             <groupId>io.micrometer</groupId>
             <artifactId>micrometer-core</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-reactive-routes</artifactId>
+            <version>2.7.5.Final</version>
```
Comment (MikeEdgar) [inline kafka-admin/pom.xml:240]: Remove version

**Review by MikeEdgar — CHANGES_REQUESTED** [review]

**Comment by MikeEdgar** [comment]
We will also need to managed the smallrye-jwt version to the current version so that we can configure the OAuth trusted certificate similar to what was previously done in `TrustedSSOCallerPrincipalFactoryProducer`. To do this, set the value of `smallrye.jwt.client.tls.certificate` to `${kafka.admin.oauth.trusted.cert:}` in the application properties. The value will be provide by kas-fleetshard in the environment variables.

https://github.com/smallrye/smallrye-jwt/releases/tag/3.5.3

File: pom.xml [hunk 2]
```diff
@@ -89,6 +89,11 @@
                 <artifactId>kafka-oauth-common</artifactId>
                 <version>${strimzi-oauth.version}</version>
             </dependency>
+            <dependency>
+                <groupId>io.quarkus</groupId>
+                <artifactId>quarkus-reactive-routes</artifactId>
+                <version>2.7.5.Final</version>
+            </dependency>
```
[the remark this answers is not in the document]
Comment (biswassri) [reply pom.xml:96]: Removal of this is giving me `The project org.bf2:kafka-admin:0.12.0-SNAPSHOT (/Users/srijoni/Desktop/Work/main/kafka-admin-api/kafka-admin/pom.xml) has 1 error
[ERROR]     'dependencies.dependency.version' for io.quarkus:quarkus-reactive-routes:jar is missing. @ line 237, column 21`

File: pom.xml [hunk 3]
```diff
@@ -26,12 +26,13 @@
         <docker.push>false</docker.push>
 
         <!-- Dependencies -->
-        <quarkus.version>2.2.5.Final</quarkus.version>
+        <quarkus.version>2.7.5.Final</quarkus.version>
         <jandex-maven-plugin.version>1.2.1</jandex-maven-plugin.version>
         <jandex.version>2.4.1.Final</jandex.version>
         <kafka.version>3.0.0</kafka.version>
         <strimzi-oauth.version>0.8.1</strimzi-oauth.version>
         <smallrye-openapi.version>2.1.20</smallrye-openapi.version>
```
Comment (MikeEdgar) [inline pom.xml:34]: No longer needed with Quarkus 2.7.5, also remove `smallrye-open-api-core` from `dependencyManagement`
```suggestion
```

File: pom.xml [hunk 4]
```diff
@@ -26,12 +26,13 @@
         <docker.push>false</docker.push>
 
         <!-- Dependencies -->
-        <quarkus.version>2.2.5.Final</quarkus.version>
+        <quarkus.version>2.7.5.Final</quarkus.version>
         <jandex-maven-plugin.version>1.2.1</jandex-maven-plugin.version>
         <jandex.version>2.4.1.Final</jandex.version>
```
Comment (MikeEdgar) [inline pom.xml:31]: No longer needed with Quarkus 2.7.5, also remove `jandex` from `dependencyManagement`
```suggestion
```

**Review by MikeEdgar — CHANGES_REQUESTED** [review]

File: pom.xml [hunk 5]
```diff
@@ -89,6 +90,11 @@
                 <artifactId>kafka-oauth-common</artifactId>
                 <version>${strimzi-oauth.version}</version>
             </dependency>
+            <dependency>
+                <groupId>io.smallrye</groupId>
+                <artifactId>smallrye-jwt-build</artifactId>
```
[the remark this answers is not in the document]
Comment (biswassri) [reply pom.xml:95]: oh sorry! Mistake.

**Review by MikeEdgar — APPROVED** [review]
LGTM :+1:

**Review by rareddy — APPROVED** [review]
lgtm

**Pull request merged**